pkg/disttask/framework/scheduler/nodes.go

```diff
@@ -185,3 +185,30 @@ func filterByScope(nodes []proto.ManagedNode, targetScope string) []string {
 	}
 	return nodeIDs
 }
+
+// SplitItemsEvenlyToWorkers split the items into batches that is suitble to be handled by `workerCnt` workers.
+func SplitItemsEvenlyToWorkers[T any](
+	items []T,
+	workerCnt int,
+	defaultStep int,
+	handleOneBatch func([]T),
+) {
+	itemLen := len(items)
+	step := defaultStep
+	if workerCnt > 0 {
```

Comment:
can you consider reuse `mathutil.Divide2Batches`, you can calculate how many subtasks to generate first, then use this method